node/cmd/guardiand/adminclient.go

```diff
@@ -168,6 +178,30 @@ func getPublicRPCServiceClient(ctx context.Context, addr string) (*grpc.ClientCo
 	return conn, c, err
 }
 
+func runSignWormchainValidatorAddress(cmd *cobra.Command, args []string) error {
+	guardianKeyPath := args[0]
+	wormchainAddress := args[1]
+	if !strings.HasPrefix(wormchainAddress, "wormhole") || strings.HasPrefix(wormchainAddress, "wormholeval") {
+		return fmt.Errorf("must provide a bech32 address that has 'wormhole' prefix")
+	}
+	gk, err := loadGuardianKey(guardianKeyPath)
+	if err != nil {
+		return fmt.Errorf("failed to load guardian key: %v", err)
```

Comment:
<i class='graphite__hidden'>[Re: lines 189 to 189]<br/></i>

Style nit: use `%w` when wrapping errors.  See [this blog post from the Go team](https://go.dev/blog/go1.13-errors) for more details.  Same comment for the lines below.<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/1963?utm_source=unchanged-line-comment">Graphite</a>.</span>